src/estimators/bloom_filters.py

```diff
@@ -501,14 +501,12 @@ def __call__(self, sketch_list):
 class SurrealDenoiser(DenoiserBase):
   """A closed form denoiser for a list of Any Distribution Bloom Filter."""
 
-  def __init__(self, epsilon=None, probability=None, flip_one_probability=None,
+  def __init__(self, epsilon=None, flip_one_probability=None,
                flip_zero_probability=None):
```

Comment:
Are "flip_zero" and "flip_one" still necessary?

Reply:
After discussing with Jiayu, we don't need flip_x_probability now. I remove it in the new commit.